### PR TITLE
templates: add placeholder for SU

### LINKED ISSUE #7401: templates/private.properties does not contain entry for geocaching.su

### PR DESCRIPTION
**Pull request by okainov**
Fixes #7401

### CODE REVIEW

File: templates/private.properties [hunk 1]
```diff
@@ -30,3 +30,8 @@ ocro.okapi.consumer.secret=
 # Opencache.uk
 ocuk.okapi.consumer.key=
 ocuk.okapi.consumer.secret=
+
+
```
Comment (UniQP) [inline templates/private.properties]: One empty line is sufficient.

**Comment by bekuno** [comment]
In the README.md are some lines where new user can find information to api keys. It would be nice also to add some lines there for gc.su

**Comment by okainov** [comment]
@bekuno done, however I'm not sure how helpful is it... Also you can ask me personally for keys..

**Comment by Lineflyer** [comment]
> I'm not sure how helpful is it... Also you can ask me personally for keys..

Maybe in a year or later or if you might not be available on Github..or whatever :)

**Comment by bekuno** [comment]
Seams fine to me.
It was for me very helpful when I started with AS and c:geo dev to find such initial information.
I used the dialog from the link. 
Мой русский больше не хорош. Школа была давным-давно. (with help from deepl)
I took therefore the english language. :-)

**Pull request merged**